django_project_base/account/rest/profile.py

```diff
@@ -184,29 +184,33 @@ class ProfileViewSet(ModelViewSet):
     pagination_class = ModelViewSet.generate_paged_loader(30, ["un_sort", "id"])
 
     def get_queryset(self):
-        qs = swapper.load_model("django_project_base", "Profile").objects.annotate(
-            un=Concat(
-                Coalesce(
-                    Case(When(first_name="", then="username"), default="first_name", output_field=CharField()),
-                    Value(""),
+        qs = (
```

Comment:
tega pa ne razumem: to je moja koda, jaz sem dodajal un_sort. A to sem dodajal nekam drugam in si sedaj ti skopiral? zakaj pa potem queryseta nisi posortiral po un_sort (vrstica 234)?